### PR TITLE
Use the two-stage developer agreement flow (bug 1208303).

### PR DESCRIPTION
**Pull request by chuckharmston**

### CODE REVIEW

**Comment by ngokevin** [comment]
gif?

**Comment by chuckharmston** [comment]
UX hasn't changed at all, really. We just get the URL for the TOS from an endpoint now.

File: src/media/js/site/components/tos.js [hunk 1]
```diff
@@ -7,6 +7,14 @@ export const tosUrl = urlJoin(process.env.MKT_ROOT,
 
 
 export default class TOSIframe extends React.Component {
+  static propTypes = {
+    url: React.PropTypes.string.isRequired,
+  };
+
+  getURL() {
```
Comment (ngokevin) [inline src/media/js/site/components/tos.js]: this isn't used is it?

plus i prefer ```getUrl``` as a super nit

**Comment by ngokevin** [comment]
r+

**Pull request merged**